examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1058,17 +1048,52 @@ def load_model_hook(models, input_dir):
                 raise ValueError(f"unexpected save model: {model.__class__}")
 
         lora_state_dict, network_alphas = LoraLoaderMixin.lora_state_dict(input_dir)
-        LoraLoaderMixin.load_lora_into_unet(lora_state_dict, network_alphas=network_alphas, unet=unet_)
 
-        text_encoder_state_dict = {k: v for k, v in lora_state_dict.items() if "text_encoder." in k}
-        LoraLoaderMixin.load_lora_into_text_encoder(
-            text_encoder_state_dict, network_alphas=network_alphas, text_encoder=text_encoder_one_
-        )
+        unet_state_dict = {f'{k.replace("unet.", "")}': v for k, v in lora_state_dict.items() if k.startswith("unet.")}
```

Comment:
Starting from Python 3.9, the `str.removeprefix` [method](https://docs.python.org/3/library/stdtypes.html#str.removeprefix) could be used.

Reply:
Thanks for the suggestion! However, the pattern I used here is used across the library. So, would prefer sticking to it for consistency.